### PR TITLE
Split --input-bundle into --input-facts-from-bundle and --input-from-bundle

### PR DESCRIPTION
**Pull request by cchudant**
Motivation #797

I did not change the CI (yet?)
Not sure this is the cleanest

I only changed the `run` command for `--input-from-bundle`. You were talking about `dump` / `compare` too but i'm not sure I understand: dump does not run the model, compare only uses random input, right?

### CODE REVIEW

**Comment by kali** [comment]
`dump` needs data for `--profile`, `compare` can use pre-determined input if you ask. And same for `bench` actually. See the various invocation of `retrieve_or_make`...

**Review by kali — APPROVED** [review]
It's not great, but good enough. The CLI code is quite far from great anyway.

**Comment by kali** [comment]
Thinking about this --input-from-bundle... It may make sense to have --allow-random-input on the subcommand as well, even if it implies fixins the CI.
We could also group these two parameters decl in a function (like the display options) which we can can apply to dump, compare, run and profile.

**Comment by cchudant** [comment]
I can add `--allow-random-input` & other args like it as a subcommand too yes, i'll try and do that

**Comment by cchudant** [comment]
Arg --const-input seems to be unused / unimplemented
https://github.com/sonos/tract/blob/328c7acfb41ababa2086b74d554d8f1356bef490/cli/src/main.rs#L90

At least, i can't find the implementation if it exists. Can it be removed?

**Comment by cchudant** [comment]
@kali I think that's it

Other arguments affected:
--allow-random-input
--allow-float-casts

Should I change the CI? The changes I made are backward-compatible.

**Comment by kali** [comment]
I am surprised about const-input. It serves when running onnx test cases from the cli (which is useful for debugging mostly). As in the onnx test manifest file, we sometimes need to say that some inputs are actually const (to avoid supporting operator configuration with variable inputs that are never used).

I may have broken it while refactoring around the TensorValues struct... It will fix it if/when I need it, I guess... It's not that often.

**Comment by kali** [comment]
Yes, I think it would be nice to fix the various invocation in the CI scripts in case somebody uses them as examples...

**Comment by cchudant** [comment]
@kali 
I changed the CI, and fixed a problem. I think this is ready to merge!

File: cli/src/main.rs [hunk 1]
```diff
@@ -99,7 +102,7 @@ fn main() -> tract_core::anyhow::Result<()> {
         .arg(arg!(--"onnx-test-data-set" [data_set] "Use onnx-test data-set as input (expect test_data_set_N dir with input_X.pb, etc. inside)"))
         .arg(arg!(--"onnx-ignore-output-shapes" "Ignore output shapes from model (workaround for pytorch export bug with mask axes)"))
 
-        .arg(arg!(--"input-node" [node] ... "Override input nodes names (auto-detects otherwise)."))
+        .arg(arg!(--"input-node" [node] ... "Oveide input nodes names (auto-detects otherwise)."))
```
Comment (kali) [inline cli/src/main.rs:105]: oupsie :)

**Review by kali — APPROVED** [review]

**Review by kali — APPROVED** [review]

**Pull request merged**